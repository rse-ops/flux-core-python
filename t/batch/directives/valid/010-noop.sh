#!/bin/sh
#
# flux: -n 32
# flux: # clear environment except for PATH and PYTHONPATH
# flux: --env=-* --env=PATH --env=PYTHONPATH
