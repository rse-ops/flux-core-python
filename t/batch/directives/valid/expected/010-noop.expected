SETARGS(['-n', '32'])
NOOP([])
SETARGS(['--env=-*', '--env=PATH', '--env=PYTHONPATH'])
